packages/webpack-cli/lib/groups/resolveConfig.js

```diff
@@ -172,7 +174,15 @@ const finalize = async (moduleObj, args) => {
 
             return rawConfig;
         }),
-    );
+    ).then((allConfigs) => {
+        allConfigs.map((singleConfig) => {
+            if (Array.isArray(singleConfig)) {
+                singleConfig.map((conf) => configs.push(conf));
+            } else {
+                configs.push(singleConfig);
+            }
+        });
```

Comment:
Nit: `map` shouldn’t be used for side effects; there’s `forEach` for that. But more simply,

```suggestion
        configs.push(...allConfigs.flat());
```

Reply:
Thanks.